examples/tutorials/Differentiation_Infrastructure_in_Deepchem.ipynb

```diff
@@ -0,0 +1,192 @@
+{
```

Comment:
<p>Can we have a short explanation of what non linear equations are and how we solve them without ML? Maybe add a graphical representation of a simple NLE</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3912/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Yes agreed. Make sure to render these in latex correctly so they show up nicely as equations on the notebook</p>